src/main/java/com/example/HR_AppJava/HR_App/HardSkill.java

```diff
@@ -1,5 +1,36 @@
 package com.example.HR_AppJava.HR_App;
 
-public class Hardskill {
-    
+import java.util.Scanner;
+
+public class HardSkill {
+
+    Scanner scanner = new Scanner(System.in);
+
+    private String title;
+    private String level;
+
+    public HardSkill(){
+        setTitle(title);
+        setLevel(level);
+    }
+
+    public void setTitle(String title){
+        System.out.print("Enter title(skill): ");
+        this.title = scanner.nextLine();
+    }
+    public void setLevel(String level){
```

Comment:
You don't need to pass a parameter to the modifier since you are getting the user input inside the method.